app/test/unit/git/remote-test.ts

```diff
@@ -32,6 +33,14 @@ describe('git/remote', () => {
       expect(result[1].name).to.equal('origin')
       expect(result[1].url.endsWith(nwo)).to.equal(true)
     })
+
+    it('returns empty array for directory without a .git directory', async () => {
+      const emptyDir = mkdirSync('no-repo-here')
+      const repository = new Repository(emptyDir, -1, null, false)
```

Comment:
should we make a fixture for this in the vein of `setupEmptyRepository`?
also same thoughts from #5299 about `before` apply

Reply:
:thumbsup: to helper function for creating the fixture